backend/attendance/CHANGELOG.md

```
@@ -1,6 +1,16 @@
-
 All notable changes to this module will be documented in this file.
 
+## 1.2.0 - 2025-03-15
+
+### Changes
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)